qa-steps/src/main/java/org/eclipse/kapua/qa/steps/RestClientSteps.java

```diff
@@ -50,16 +62,22 @@ public void setHostPort(String host, String port) {
         stepData.put("port", port);
     }
 
-    @When("^REST call at \"(.*)\"")
-    public void restCallStatusOfIndex(String resource) throws Exception {
+    @When("^REST GET call at \"(.*)\"")
+    public void restGetCall(String resource) throws Exception {
 
         String host = (String) stepData.get("host");
         String port = (String) stepData.get("port");
+        String tokenId = (String) stepData.get("tokenId");
         try {
             URL url = new URL("http://" + host + ":" + port + resource);
             HttpURLConnection conn = (HttpURLConnection) url.openConnection();
```

Comment:
Please use `try with resources` to avoid streams to left open in case of failures